src/sqlfluff/cli/commands.py

```diff
@@ -868,6 +875,19 @@ def fix(
             click.echo(f"=== {step} ===")
             click.echo(formatter.cli_table(timing_summary[step].items()))
 
+    if show_lint_violations:
+        click.echo("==== lint for unfixable violations ====")
+        lnt, formatter = get_linter_and_formatter(config)
+        for violation in result.get_violations(**num_violations_kwargs):
```

Comment:
Does this lint the file a second time? I wonder if there's a way to get the list of unfixable violations without doing this. This will basically cause the run time to double.

Reply:
the formatter was duplicate so i removed it.
the actual parsing is done on line 796 (result = lnt.lint_paths) and the result is in result, so the get_vliolations does not reparse the file, only the result set